catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSourceTest.java

```diff
@@ -1670,75 +1672,111 @@ public void testSortingDisabled() throws Exception {
         new QueryImpl(builder.attribute(Metacard.ANY_TEXT).is().like().text("literal"));
-    setupMapper(null, null, null);
+    setupMapper(
+        MOCK_TEMPORAL_SORT_PROPERTY, MOCK_RELEVANCE_SORT_PROPERTY, MOCK_DISTANCE_SORT_PROPERTY);
     source.setMetacardMappers(metacardMappers);
-    source.setDisableSorting(false);
-    propertyIsLikeQuery.setSortBy(new SortByImpl("title", SortOrder.ASCENDING));
+    propertyIsLikeQuery.setSortBy(new SortByImpl(null, "ASC"));
+
+    ArgumentCaptor<ExtendedGetFeatureType> argumentCaptor =
+        ArgumentCaptor.forClass(ExtendedGetFeatureType.class);
 
     source.query(new QueryRequestImpl(propertyIsLikeQuery));
+
+    verify(mockWfs, times(2)).getFeature(argumentCaptor.capture());
+
+    assertThat(argumentCaptor.getAllValues().get(1).getQuery().get(0).getSortBy(), is(nullValue()));
   }
 
   @Test
-  public void testSortingNoSortOrder() throws Exception {
-    // if sort order is missing, throw UnsupportedQueryException
-    expectedEx.expect(UnsupportedQueryException.class);
-    expectedEx.expectMessage("Source WFS_ID does not support specified sort property TEMPORAL");
-
+  public void testNullSortProperty() throws Exception {
     mapSchemaToFeatures(ONE_TEXT_PROPERTY_SCHEMA_PERSON, ONE_FEATURE);
     setUpMocks(null, null, ONE_FEATURE, ONE_FEATURE);
     final QueryImpl propertyIsLikeQuery =
         new QueryImpl(builder.attribute(Metacard.ANY_TEXT).is().like().text("literal"));
     setupMapper(
         MOCK_TEMPORAL_SORT_PROPERTY, MOCK_RELEVANCE_SORT_PROPERTY, MOCK_DISTANCE_SORT_PROPERTY);
     source.setMetacardMappers(metacardMappers);
-    propertyIsLikeQuery.setSortBy(new SortByImpl(Result.TEMPORAL, (String) null));
+    propertyIsLikeQuery.setSortBy(new SortByImpl(new PropertyNameImpl(null), null));
 
     source.query(new QueryRequestImpl(propertyIsLikeQuery));
   }
 
   @Test
-  public void testSortingNoSortProperty() throws Exception {
-    // query is still valid even if sort property is missing
+  public void testSortingBadSortOrderWithoutDefault() throws Exception {
+
+    // query is still valid even if sort property bad
     mapSchemaToFeatures(ONE_TEXT_PROPERTY_SCHEMA_PERSON, ONE_FEATURE);
     setUpMocks(null, null, ONE_FEATURE, ONE_FEATURE);
     final QueryImpl propertyIsLikeQuery =
         new QueryImpl(builder.attribute(Metacard.ANY_TEXT).is().like().text("literal"));
     setupMapper(
         MOCK_TEMPORAL_SORT_PROPERTY, MOCK_RELEVANCE_SORT_PROPERTY, MOCK_DISTANCE_SORT_PROPERTY);
     source.setMetacardMappers(metacardMappers);
-    propertyIsLikeQuery.setSortBy(new SortByImpl(null, "ASC"));
+    propertyIsLikeQuery.setSortBy(new SortByImpl(Result.TEMPORAL, "foo"));
 
-    source.query(new QueryRequestImpl(propertyIsLikeQuery));
+    ArgumentCaptor<ExtendedGetFeatureType> argumentCaptor =
+        ArgumentCaptor.forClass(ExtendedGetFeatureType.class);
+
+    SourceResponse sourceResponse = source.query(new QueryRequestImpl(propertyIsLikeQuery));
+    assertThat(
```

Comment:
✏️ You don't need to change this, but I will point out that Hamcrest has a `hasEntry()` matcher: https://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/collection/IsMapContaining.html